### PR TITLE
Book a visit e2e test

### PR DESCRIPTION
**Pull request by Vamsi-QA**

### CODE REVIEW

**Comment by Vamsi-QA** [comment]
**What does this PR do?**
This is a 'Happy Path'  Book a visit  End-to-end staff UI test using Playwright

File: src/pages/LoginPage.ts [hunk 1]
```diff
@@ -16,10 +17,30 @@ export default class LoginPage extends BasePage {
         await this.page.context().storageState({ path })
     }
 
-    async signInWith(userName: string, passWord: string): Promise<void> {
-        await this.inputUserName.fill(userName)
-        await this.inputPassword.fill(passWord)
+    async signInWith(userName: UserType): Promise<void> {
+        await this.inputUserName.fill(await this.getUserName(userName))
+        await this.inputPassword.fill(process.env.PASSWORD || '')
```
Comment (adaviesMOJ) [inline src/pages/LoginPage.ts:22]: Will this attempt to sign in without a password? Is there a "Fail fast" option, if process.env.PASSWORD null -> exit 1 kinda thing?

**Review by adaviesMOJ — APPROVED** [review]
LGTM - only 1 question

**Review by tpmcgowan — APPROVED** [review]
LGTM!

**Pull request merged**